Description: remove setuptools_scm dependency

* Be careful with the tabs

Index: wazo-python-tempora-packaging/setup.py
===================================================================
--- wazo-python-tempora-packaging.orig/setup.py
+++ wazo-python-tempora-packaging/setup.py
@@ -14,7 +14,7 @@ pkg_resources "managed" namespace packag
 
 params = dict(
 	name=name,
-	use_scm_version=True,
+	version="1.13",
 	author="Jason R. Coombs",
 	author_email="jaraco@jaraco.com",
 	description=description or name,
@@ -53,7 +53,6 @@ params = dict(
 		],
 	},
 	setup_requires=[
-		'setuptools_scm>=1.15.0',
 	],
 	classifiers=[
 		"Development Status :: 5 - Production/Stable",
